XenAdmin/Controls/Ballooning/HostMemoryRow.cs

```diff
@@ -64,5 +65,16 @@ internal void UnregisterHandlers()
             memoryRowLabel.UnsubscribeEvents();
             hostMemoryControls.UnregisterHandlers();
         }
+
+        public MetricUpdater MetricUpdater

```

Comment:
I think we should have the MetricUpdater as a read-only property in this class, add a metricUpdater parameter to the constructor, where we assign it directly to hostMemoryControls.MetricUpdater